packages/expo-sensors/src/Pedometer.ts

```diff
@@ -31,6 +31,10 @@ export type PedometerUpdateCallback = (result: PedometerResult) => void;
  * provided with a single argument that is [`PedometerResult`](#pedometerresult).
  * @return Returns a [`Subscription`](#subscription) that enables you to call
  * `remove()` when you would like to unsubscribe the listener.
+ *
+ * > Pedometer updates will not be delivered while the app is in the background. As an alternative on iOS, the `getStepCountAsync`
+ * > method can be used to get the step count between two dates. On Android, consider using another solution based on
+ * > [`Health Connect API`](https://developers.google.com/fit) or [`Google Fit API`](https://developer.android.com/health-and-fitness/guides/health-connect).
```

Comment:
```suggestion
 * > Pedometer updates will not be delivered while the app is in the background. As an alternative, on Android, use another solution based on
 * > [`Health Connect API`](https://developers.google.com/fit) or [`Google Fit API`](https://developer.android.com/health-and-fitness/guides/health-connect). On iOS, the `getStepCountAsync`
 * > method can be used to get the step count between two dates.
```

Let's use the correct [platform order](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web) as per our writing styleguide.